projects/observability/src/shared/dashboard/widgets/topology/topology-widget-renderer.component.test.ts

```diff
@@ -999,15 +999,9 @@ describe('Topology Widget renderer', () => {
       }
     ];
     const spectator = createComponent();
-    const nodeRendererDestroySpy = spyOn(
-      spectator.inject(TopologyNodeRendererService, true),
-      'destroyNode'
-    ).and.callThrough();
+    const nodeRendererDestroySpy = jest.spyOn(spectator.inject(TopologyNodeRendererService, true), 'destroyNode');
```

Comment:
call through not needed?

Reply:
> By default, jest.spyOn also calls the spied method. 
Ref: https://jestjs.io/docs/jest-object#jestspyonobject-methodname